content/en/docs/shared_services/supportapi/formats/create_case_req.md

```diff
@@ -11,13 +11,15 @@ description: Request format for the Create Case method.
 | Property Name | Data Type                     | Optional | Description |
 |---------------|-------------------------------|----------|-------------|
 | sac           | string                        |       no | Support access code. |
+| contact       | string                        |       no | The e-mail of the contact. The 'contact' property is required for requests that use the [OAuthApplication security scheme](https://sphereapi.admin.axway.com/sphere/api/v1/). |
 | subject       | string                        |       no | Brief description of this case. |
 | description   | string                        |       no | Detailed description of this case. |
 | product       | [Product](#product)           |       no | Affected product. |
 | environment   | [Environment](/docs/shared_services/supportapi/formats/environment) |       no | Environment for which this case is created. |
 | impact        | [Impact](#impact)             |       no | Impact level. |
 | urgency       | [Urgency](/docs/shared_services/supportapi/formats/urgency)         |       no | Urgency level. |
-| type          | [CaseType](#casetype)         |      yes | The type of this case. Default: `Support Case`. |
+| type          | [CaseType](/docs/shared_services/supportapi/formats/case_type)         |      yes | The type of this case. Default: `Support Case`. |
+| targetDate    | string                        |      yes | Applicable when the case is a business service request. Sending 'targetDate' with other case types results in an error. See more details [here](/docs/shared_services/supportapi/formats/target_date). |
```

Comment:
Regarding the quoted property name 'targetDate, please my other comment for 'contact.
Again, this comment does not require any action, IMO.